manifests/install.pp

```diff
@@ -105,7 +105,13 @@
             }
 
             yumrepo { 'grafana':
-              descr    => 'grafana repo',
+              ensure => 'absent',
+              descr  => 'grafana repo',
+              before => Package[$::grafana::package_name],
```

Comment:
please do not prefix variables with `::` after `$`. that is legacy puppet3 syntax

Reply:
This will be fixed, after #164 has been merged, because it will introduce the necessary lint check.